cytominer_eval/tests/test_evaluate.py

```diff
@@ -160,7 +156,7 @@ def test_evaluate_precision_recall():
             replicate_groups=["Metadata_broad_sample"],
             operation="precision_recall",
             similarity_metric="pearson",
-            precision_recall_k=k,
+            precision_recall_k=[k],
```

Comment:
done

Reply:
Can you add a quick comment on line 151 noting the difference between evaluate on line 152 and evaluate on line 137?

For future changes to this code, it will be important to note that we're also testing the list/int capability of this argument